src/pages/bulkImport/utils/useBulkImportFields.js

```diff
@@ -32,6 +32,13 @@ const utcOffsetValidator = {
   error: 'Not a valid timezone (UTC offset format)',
 };
 
+function deriveLabel(field, intl) {
+  const label = field?.labelId
+    ? intl.formatMessage({ id: field.labelId })
+    : field.label;
```

Comment:
If `field?.labelId` can be null because field can be null, then this could still error.

Reply:
thanks. that wasn't the original issue but still worth fixing.

Reply:
@Emily-Ke pushed

Reply:
As, in the bottom part should be:

`field?.label `
as well?

Or does:
`null ? something: somethingElse `
inherently cause an error?

Reply:
the first one! `?.`

Reply:
I'm seeing from the new commit that it's the former.